Assets/Scripts/Projects/ProjectData.cs

```diff
@@ -199,15 +209,18 @@ public void ProjectSavedToIndexedDB()
 
         public void AddStandardLayer(LayerNL3DBase layer)
         {
-            layer.Root = RootLayer;
+            if (!isLoading)
```

Comment:
This sounds confusing to me; a StandardLayer cannot be added while the project is not loading; but the event is triggered nonetheless? Is it necessary to block adding this?

Reply:
I added a comment to the isLoading field, I can move this to this if statement if you want?
 //is the project data currently loading? if true don't add the Layers to the root's childList, because this list is stored in the json, if false, a layer was created in app, and it should be initialized

Reply:
Ah no, that's good. Sometimes reviewing gives a very narrow focus on what you see in literal changes and miss the bigger picture